xbmc/pvr/dialogs/GUIDialogPVRGuideInfo.cpp

```diff
@@ -123,6 +137,21 @@ bool CGUIDialogPVRGuideInfo::OnClickButtonRecord(CGUIMessage &message)
   return bReturn;
 }
 
+bool CGUIDialogPVRGuideInfo::OnClickButtonAddTimer(CGUIMessage &message)
+{
+  bool bReturn = false;
+
+  if (message.GetSenderId() == CONTROL_BTN_ADD_TIMER)
+  {
+    if (!m_progItem->Timer())
```

Comment:
not 100% sure; so lets put in a nullptr check here, like done in other places in this file.